bench-tps/src/bench_tps_client.rs

```diff
@@ -93,6 +94,16 @@ pub trait BenchTpsClient {
     ) -> Result<Account>;
 
     fn get_multiple_accounts(&self, pubkeys: &[Pubkey]) -> Result<Vec<Option<Account>>>;
+
+    fn get_slot(&self) -> Result<Slot>;
```

Comment:
are we ok with letting this use the `Self::commitment()` level? or should this be `get_slot_with_commitment()`. Not sure all the use cases you're looking for. But what you currently have is probably the simplest, just not sure if we need the flexibility...

Reply:
Good question since  I use a particular commitment level to fetch the blocks in the code of the main PR